src/mcts/params.cc

```diff
@@ -87,6 +87,8 @@ const OptionId SearchParams::kTemperatureVisitOffsetId{
     "Adjusts visits by this value when picking a move with a temperature. If a "
     "negative offset reduces visits for a particular move below zero, that "
     "move is not picked. If no moves can be picked, no temperature is used."};
+const OptionId SearchParams::kTerminatorId{
+    "terminator", "Terminator", "I'll be back!"};
```

Comment:
I'd write a better help for it. :) And after the help is written and it's clear what that option actually does, possibly it will also make sense to rename the option too.

Reply:
What do we want this to be called? Was waiting on your feedback before putting in actual values.

Reply:
Also, I wasn't sure if we wanted this as an actual param or just for testing the PR.

Reply:
I thought we were going to merge it (with default off at first at least for training).
Maybe running speed test on random backend makes sense to be sure it doesn't slow things down, but from the code it doesn't seem that it should.

As `Certainty Propagation` is reserved for a more global changes, something like...
`TerminalBubble` (so that "terminals bubble up from their nodes to parent")
`TerminalSpill`  (so that "terminals spill down from their nodes to parent"?
`StickyTerminals` (just sounds fine)
`TerminalPropagation` (or `PropagateTerminals`) (to make it similar to `Certainty` but not really the same)
Something like that?

Reply:
`StickyEndgames` ?

Reply:
Fine. :)
Although "endgame" in chess means something different, terminal can also happen in opening and midgame. `StickyGameEndging`?.. meh :)
`StickyOutcome`... `StickyEnd` :) Maybe `StickyNodes`? Not relevant, but sounds nice.

But I'm fine with endgames too!